### ISSUE TITLE
Konfigurace funkce Export (obsah roletky)

### ISSUE THREAD

**Issue by ZdenkaSera** [proarc/proarc-client#330]
Prosíme o vyjasnění, zda/jak můžeme konfigurací v klientovi ovlivnit seznam exportů relevantních pro jednotlivé modely. Děkujeme.

**Comment by SykoraLukas** [comment]
Při psani funkce na API jsem narazil na toto a potřebuji vyjasnit.

CROSSREF jde delat jen na ndk (a ndke) periodikach a jeho castech a eclanku. Ale CEJSH jde delat na vsech modelech ndk (a ndke) vcetne monografii a jejich castech. Je to spravne? Nebo to maji opet byt jen periodika a jeho casti?

**Comment by ZdenkaSera** [comment]
@SykoraLukas Předám kolegyni z OZČ, díky.

**Comment by SykoraLukas** [comment]
Na API je nasazeno - zatím funkcionalita je zcela stejná jako v jádře.

**Comment by ZdenkaSera** [comment]
Prosím ještě o radu, z konfigu na wiki si nejsem jistá - lze v konfiguraci ovlivnit nastavení pro tentýž model a různé instance? Konkrétně např. pro NDK mono je k dispozici KWIS, což nepoužíváme. Děkuji.

**Comment by SykoraLukas** [comment]
Navrženo to máme tak, ze Api pošle vše co pro daný model jde, ale klient to ještě vyfiltruje podle toho, co má v konfigu (necha jen ty hodnoty, co jsou z API tak v konfigu)

**Comment by albertoh** [comment]
Nasazeno na testu

**Comment by SykoraLukas** [comment]
Ano, Crossref nad emonografii to zatím neumí. Takže zatím tuto podmínku mažu a až bude požadavek na vývoj tak to povolíme a uděláme export pro Emonografii.

**Comment by SykoraLukas** [comment]
Změna nasazena.

**Comment by ZdenkaSera** [comment]
Souvisí s https://github.com/proarc/proarc-client/issues/510

**Comment by tumovam** [comment]
[Tabulka ](https://docs.google.com/spreadsheets/d/1dTwirnEImuZ5WNGPUFtyLIIQa_T1yeqPeByFuF3SkpU/edit?usp=sharing)modely a exporty. Zbytek v globals - [nasazení klienta](https://github.com/proarc/proarc-client/wiki/Nasazen%C3%AD-klienta).
